pkg/linode-bs/nodeserver.go

```diff
@@ -72,10 +77,21 @@ func (ns *LinodeNodeServer) NodePublishVolume(ctx context.Context, req *csi.Node
 	if len(targetPath) == 0 {
 		return nil, status.Error(codes.InvalidArgument, "NodePublishVolume Target Path must be provided")
 	}
+	if len(targetPathDir) == 0 {
```

Comment:
i think this can be dropped. this doesn't actually validate anything as the error says and the targetPath check already confirms the param was passed. additionally, the mkdir below will error if the path was bad so it's technically validated that way

Reply:
No problem.